Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -72,15 +88,21 @@ def __init__( self, ceUniqueID ):
     self.outputURL = 'gsiftp://localhost'
     self.gridEnv = ''
     self.proxyRenewal = 0
+    self.extraSubmitString = self.ceParameters.get('ExtraSubmitString', '').decode('string_escape')
+
+    ## see note on getCondorLogFile, why we can only use the global setting
+    self.workingDirectory = gConfig.getValue( "Resources/Computing/HTCondorCE/WorkingDirectory",
```

Comment:
So in fact you were referring to the problem addressed in the FIXME in getCondorLogFile function. So I can create a (HTCondorCE)ComputingElement instance in the getCondorLogFile function or I leave it to you to change the WMSAdministrator to create ComputingElement instances instead?